Part2-API-Schemas/openapi.yaml

```diff
@@ -638,17 +622,19 @@ components:
         - $ref: "#/components/schemas/SubmodelElementAttributes"
       properties: 
         valueType: 
-          $ref: "https://api.swaggerhub.com/domains/Plattform_i40/Part1-MetaModel-Schemas/V3.0.1#/components/schemas/DataTypeDefXsd"
+          $ref: "https://api.swaggerhub.com/domains/Plattform_i40/Part1-MetaModel-Schemas/V3.0.2#/components/schemas/DataTypeDefXsd"
     RangeValue: 
       properties: 
         max: 
-          type: number
+          $ref: "#/components/schemas/RangeValueType"
         min: 
-          type: number
-      required: 
-        - min
-        - max
+          $ref: "#/components/schemas/RangeValueType"
       type: object
+    RangeValueType:
+      oneOf:
+        - $ref: "#/components/schemas/StringValue"
+        - $ref: "#/components/schemas/NumberValue"
+        - $ref: "#/components/schemas/BooleanValue"
```

Comment:
In case of date range it could help to be more precise and define a DateValue:

    DateValue: 
      type: string
      format: date-time
      
Without that every date format could be used and would be be not interoperable.

Reply:
Hi Sven, this would be a topic for https://github.com/admin-shell-io/aas-specs and then also for the next minor (major?) release version. Therefore, we won't change it in this PR.

Reply:
I am not sure this would be backward compatible to include it in V3.1 but I added a corresponding issue: https://github.com/admin-shell-io/aas-specs/issues/444

Reply:
The addition of string was part of the API bugfix für ValueOnly, it was not there before, only numerical. So it would be possible to also add DateValue as part of the bugfix.